www/contribute/wanted-ebooks.php

```diff
@@ -559,9 +559,6 @@
 		</ul>
 		<h2>Advanced productions</h2>
 		<ul>
-			<li>
```

Comment:
Why are we removing this?

Reply:
Again, my apologies, I don't know why these two elements were deleted in this commit. Quite possibly deleted by my kitten walking on the keyboard when I wasn't looking. She's not great at HTML.
Will update or replace the PR with a correct commit.